backend/src/main/java/org/globe42/web/users/PasswordGenerator.java

```diff
@@ -0,0 +1,37 @@
+package org.globe42.web.users;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+import org.springframework.stereotype.Component;
+
+/**
+ * A random password generator
+ * @author JB Nizet
+ */
+@Component
+public class PasswordGenerator {
+
+    private static final int SIZE = 8;
+    private static final char[] CHARACTERS =
+        "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_$%?/+=.<>#*".toCharArray();
```

Comment:
indicate perhaps in the javadoc that it's an array initialized with all the accepted chars